x-pack/plugins/maps/public/layers/vector_layer.js

```diff
@@ -641,7 +641,7 @@ export class VectorLayer extends AbstractLayer {
     }
   }
 
-  _setMbPointsProperties(mbMap) {
+  _setMbPointsProperties(mbMap, options) {
```

Comment:
Why are you wrapping mvtSourceLayer in an options object? Do you see additional options properties? In the past, these nebulas option objects get confusing and its hard to remember why something goes in the options vs is just an additional argument.

I would recommend either 
`_setMbPointsProperties(mbMap, mvtSourceLayer)`
`_setMbPointsProperties({ mbMap, mvtSourceLayer })`